com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/HDShadowManager.RenderGraph.cs

```diff
@@ -36,7 +36,7 @@ internal static ShadowResult ReadShadowResult(in ShadowResult shadowResult, Rend
         internal void RenderShadows(RenderGraph renderGraph, in ShaderVariablesGlobal globalCB, HDCamera hdCamera, CullingResults cullResults, ref ShadowResult result)
         {
             // Avoid to do any commands if there is no shadow to draw
-            if (m_ShadowRequestCount != 0)
+            if (m_ShadowRequestCount != 0 && hdCamera.frameSettings.IsEnabled(FrameSettingsField.OpaqueObjects))
```

Comment:
I am not sure I understand this one, we can render transparent objects inside shadow, we cast shadow from them

Reply:
Missing an additional check, that's why :)